script.service.ltvpl/addon.xml

```diff
@@ -0,0 +1,57 @@
+<addon id="script.service.ltvpl" name="Live TV Playlist" provider-name="jmooremcc" version="1.0.0">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0" />
+    <import addon="xbmc.addon" version="17.0.0" />
+    <import addon="xbmc.gui" version="5.14.0" />
+  </requires>
+  <extension library="service.py" point="xbmc.service" />
+  <extension library="resources/lib" point="xbmc.python.module" />
+  <extension library="resources" point="xbmc.python.module" />
```

Comment:
neither should be needed. this exposes the entire resource dir as a python library to the outside world.
the xbmc.python.module extension point is intended for script.module.* addons.

the only exception would be if you need to share a part of your code with another addon, but i don't think that's the case here.

Reply:
When I remove `<extension library="resources" point="xbmc.python.module" />` from my addon.xml file, none of my textures  appear when the gui window is displayed. 
The only way I know how to make it work is by including that line in my addon.xml file.
How does Kodi know where to search for my textures?
Is there another way for me to tell Kodi where my textures are?